.gitignore

```diff
@@ -1,3 +1,8 @@
+.DS_Store
+*.iml
+.classpath
+.project
+.idea/
```

Comment:
I'm completely at your side. And I'd love to develop here with IntelliJ, but did not find a good way to get everything up and running inside of it.

I just said that to keep review-cycles down. And maybe the strategy to just ignore one IDE files has changes.